src/hosted_agents/hosted_agents_starter.js

```diff
@@ -26,3 +24,13 @@ function register_rpc() {
         logging: true,
     });
 }
+
+async function start_hosted_agents() {
```

Comment:
This is a function that is called from the global scope who cannot wait so waiting or not waiting does not make a difference. 

Before the change all of the calls where separately on the global scope (again without the ability to wait) and I did not change it, as I wrote above, it will be meaningless because we have nothing to do after we wait.

If you insist I can add the wait

Reply:
If all I need to do is insist then I insist :)
My point it that async functions better be awaiting promises and not spread them around careleslly.

Reply:
And of course it wans't your PR that created this code, but I'm trying to align that code to be more predictable as you also desire.

Reply:
Done, waiting for all now

Reply:
La Insistance! :) Resolving.